src/objective-c/tests/GRPCClientTests.m

```diff
@@ -200,7 +201,7 @@ - (void)testMetadata {
   request.fillOauthScope = YES;
   GRXWriter *requestsWriter = [GRXWriter writerWithValue:[request data]];
 
-  GRPCCall *call = [[GRPCCall alloc] initWithHost:kHostAddress
+  GRPCCall *call = [[GRPCCall alloc] initWithHost:kRemoteSSLHost
```

Comment:
So is this test enabled now? If so, please remove the TODO on top of the method.